libs/components/indicators/src/lib/modules/help-inline/help-inline.component.scss

```diff
@@ -15,20 +17,23 @@
   }
 }
 
-:host-context(.sky-control-help) {
```

Comment:
Removed this based on a conversation with @Blackbaud-AdamFunderburk. We want all help inline buttons to have the same padding no matter where they are being constructed. This was removing the padding in places like Box and Tiles previously.